### ISSUE TITLE
740 added entry--uncontrolled related/analytical title

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#262]
https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/7XX.csv

**Comment by lake44me** [comment]
@cspayne are we using the "nonfiling indicators" in any way, in transformation?  I am specifying unstructured datatype objects for subfields, a, n, and p in 740 which is an uncontrolled title.  I would like to simplify the mappings to ignore indicator 1 values.

**Comment by lake44me** [comment]
A note for whoever reviews this - I decided to map the obsolete indicator 2 values (0, 1, 3).  A search of orbis cascade records using https://libweb.library.wwu.edu/ils/apps/marc_search/search.php and https://orbiscascade-washington.primo.exlibrisgroup.com/discovery/search?vid=01ALLIANCE_UW:UW showed a large number of records with 0 and 1.  I made a best guess as to the meaning of these indicators based on a sampling of what I found.  Indicator 3 seems to only have been used for VM format; I couldn't find any examples but decided to map it anyway.  Choices seem to boil down to variant title of manifestation or related work of manifestation (the least specific property I could find).

**Comment by lake44me** [comment]
@sarahruska, I don't think that would be a good idea.  The label for this string would be supplied by the system based on the property used. whether "has variant title of manifestation", "has variant title of item", "has related work of manifestation" etc.  The fact that we refer to a related work by its title is an RDA thing that user interface builders may choose to clarify by adding Title:  to the label, but I don't think we should get into making prefixes that may duplicate that or complicate the choice of labeling in different languages, etc.  Sort of like that MARC note - it's something a system could add later.  I suspect that note is a vestige of something MARC used to do that was revised out.  

Make sense?

**Comment by sarahruska** [comment]
That makes sense! Thanks, Laura

**Comment by Sarahanc** [comment]
@cspayne I'm not quite sure how to implement the mapping regarding testing for aggregates. Do I need to use an additional template in the code for this field to test for aggregates? Or will aggregate testing occur outside of the field-by-field code? Thank you!
